package.json

```diff
@@ -1,42 +1,61 @@
 {
-	"name": "remap-istanbul",
-	"version": "0.6.5-pre",
-	"description": "A tool for remapping Istanbul coverage via Source Maps",
-	"homepage": "http://dojotoolkit.org",
-	"keywords": [ "gulpplugin", "gruntplugin", "source-map", "istanbul", "coverage" ],
-	"maintainers": [
-		{
-			"name": "kitsonk",
-			"email": "me@kitsonkelly.com"
-		}, {
-			"name": "bryanforbes",
-			"email": "bryan@reigndropsfall.net"
-		}
-	],
-	"bugs": {
-		"url": "https://github.com/SitePen/remap-istanbul"
-	},
-	"license": "BSD-3-Clause",
-	"main": "main.js",
-	"bin": "./bin/remap-istanbul",
-	"repository": {
-		"type": "git",
-		"url": "https://github.com/SitePen/remap-istanbul.git"
-	},
-	"scripts": {
-		"test": "tests/run.sh"
-	},
-	"dependencies": {
-		"amdefine": "1.0.0",
-		"gulp-util": "3.0.7",
-		"istanbul": "0.4.5",
-		"source-map": ">=0.5.6",
-		"through2": "2.0.1"
-	},
-	"devDependencies": {
-		"codecov.io": "0.1.6",
-		"grunt": "^1.0.1",
-		"gulp": "3.9.1",
-		"intern": "^3.3.0"
-	}
+  "name": "remap-istanbul",
+  "version": "0.6.5-pre",
+  "description": "A tool for remapping Istanbul coverage via Source Maps",
+  "homepage": "http://dojotoolkit.org",
+  "keywords": [
+    "gulpplugin",
+    "gruntplugin",
+    "source-map",
+    "istanbul",
+    "coverage"
+  ],
+  "maintainers": [
+    {
+      "name": "kitsonk",
+      "email": "me@kitsonkelly.com"
+    },
+    {
+      "name": "bryanforbes",
+      "email": "bryan@reigndropsfall.net"
+    }
+  ],
+  "bugs": {
+    "url": "https://github.com/SitePen/remap-istanbul"
+  },
+  "license": "BSD-3-Clause",
+  "main": "./lib/main.js",
+  "bin": "./lib/remap-istanbul.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/SitePen/remap-istanbul.git"
+  },
+  "scripts": {
+    "lint": "eslint ./src/",
+    "test": "tests/run.sh",
+    "build": "rimraf lib && babel --presets es2015 --plugins transform-runtime src --out-dir ./lib --source-maps",
+    "prepublish": "npm run build"
+  },
+  "dependencies": {
+    "amdefine": "1.0.0",
+    "babel-runtime": "^6.11.6",
+    "gulp-util": "3.0.7",
+    "istanbul": "0.4.5",
+    "source-map": ">=0.5.6",
+    "through2": "2.0.1"
+  },
+  "devDependencies": {
+    "babel-cli": "^6.16.0",
+    "babel-core": "^6.17.0",
+    "babel-plugin-transform-runtime": "^6.15.0",
+    "babel-preset-es2015": "^6.16.0",
+    "codecov.io": "0.1.6",
+    "eslint": "^3.7.0",
+    "eslint-config-airbnb-base": "^8.0.0",
+    "eslint-plugin-import": "^1.16.0",
+    "grunt": "^1.0.1",
+    "gulp": "3.9.1",
+    "intern": "^3.3.0",
+    "rimraf": "^2.5.4"
```

Comment:
so what was added:

1. babel and plugins for it, to target older nodejs versions
2. eslint and presets for it to check codestyle
3. rimraf - to clean dist folder before building new version (to prevent accidentally pushing obsolete files to npm)